app/controllers/api/v1/categories_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::CategoriesController < ApplicationController
+  def index
+    @categories = Category.all
+    render json: @categories.map { |user| user.as_json only: :name }
```

Comment:
@moizansari22 Kindly explain me what the purpose of this endpoint and what's it returning?

Reply:
the basic purpose of this endpoint is to just return the names of categories which we have in our database 
<img width="864" alt="Screenshot 2022-10-14 at 4 31 04 PM" src="https://user-images.githubusercontent.com/113978056/195841719-1ddf7ff8-6981-4f52-9dfe-720d430b05c5.png">

Reply:
This is wrong approach.
```suggestion
    render json: @categories.pluck(:id, :name)
```